federatedscope/core/configs/cfg_fl_setting.py

```diff
@@ -67,10 +67,10 @@ def extend_fl_setting_cfg(cfg):
     # ---------------------------------------------------------------------- #
     # Vertical FL related options (for demo)
     # ---------------------------------------------------------------------- #
+    cfg.vertical_dims = [5, 10]  # Avoid to be removed when `use` is False
```

Comment:
Why we still need `vertical_dims` if `vertical.use=False`?

Reply:
so what is this? I guess it is the input dim. If it is the case, why do we need it?